mthree/mitigation.py

```diff
@@ -222,20 +144,35 @@ def cals_from_system(self, qubits=None, shots=None, method='balanced',
         Raises:
             M3Error: Called while a calibration currently in progress.
         """
+        from .calibration import grab_additional_cals
+
         if self._thread:
             raise M3Error('Calibration currently in progress.')
         if qubits is None:
             qubits = range(self.num_qubits)
         self.cal_method = method
         self.rep_delay = rep_delay
         self.cal_timestamp = None
-        self._grab_additional_cals(qubits, shots=shots,  method=method,
+        grab_additional_cals(self, qubits, shots=shots,  method=method,
                                    rep_delay=rep_delay, initial_reset=initial_reset)
         if cals_file:
             self.cals_to_file(cals_file)
 
+    def cals_from_matrices(self, matrices):
```

Comment:
This kind of exposes the internal storage format which is not really in the docs. Should explain this somewhere if exposing.

Reply:
I can do this.

Reply:
Thanks, I think it'll be best if you do.